src/connection/WalletConnectV2.ts

```diff
@@ -32,6 +32,7 @@ export class WalletConnectV2Popup extends WalletConnect {
         chains: [SupportedChainId.MAINNET],
         showQrModal: qrcode,
         rpcMap: RPC_URLS_WITHOUT_FALLBACKS,
+        optionalMethods: ['eth_signTypedData', 'eth_signTypedData_v4', 'eth_sign'],
```

Comment:
Can you precede this line with a comment linking to the docs explaining why this is needed?

Reply:
unfortunately there are no docs w/r/t this configuration option, but I'm happy to add a comment with what background info we do have